Tests/test_PDB_NeighborSearch.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 by Peter Cock.  All rights reserved.
```

Comment:
This isn't my work.

The original test was written by Thomas Hamelryck (CVS username ``nisse``) in Dec 2002, see https://github.com/souravsingh/biopython/commit/b9996bf27c6c69e22ce1c78a18f5285ebf8b3fe0

I would suggest:

```
# Copyright 2017 by Peter Cock.  All rights reserved.
# Based on code Copyright 2002 by Thomas Hamelryck.  All rights reserved.
```